crates/rethnet_rpc_server/Cargo.toml

```diff
@@ -5,12 +5,15 @@ edition = "2021"
 
 [dependencies]
 axum = "0.6.18"
+bytes = { version = "1.2.1", default-features = false }
```

Comment:
```suggestion
```

This is already a public re-export of rethnet_eth, ensuring versions don't go out of sync. So no need to add it here